week-2/interpret/invert.js

```diff
@@ -10,18 +10,33 @@ function invert(obj) {
   const invertedObj = {};
 
   for (const [key, value] of Object.entries(obj)) {
-    invertedObj.key = value;
+    invertedObj[value] = key;
   }
 
   return invertedObj;
 }
 
+console.log(invert({ x: 10, y: 20 }));
+
 // a) What is the current return value when invert is called with { a : 1 }
 
+// { a : 1 }
```

Comment:
Hmm, check this again please.

Reply:
It says current? So should it be invert({a : 1}); ? <-- invert is the function name, not sure if it is actually called this

Reply:
My thought process behind this is that it's the current output and not the target output?